### ISSUE TITLE
Add support to delete mobile bookmark folder and pc bookmark folder (both root folders) at once. If not posible to delete the root folders. ADD A MENU AND A OPTION TO SELECT ALL BOOKMARKS SO THAT THEY CAN BE DELETED AT ONCE

### ISSUE THREAD

**Issue by tathastu871** [brave/brave-browser#18766]
## Steps to Reproduce

   1. CLICK ON MENU
   2. CLICK ON Bookmarks
   3. Click on back arrow in upper left corner
   4. See there is no option to select mobile bookmark folder and delete it completely 
   5. We have to select each bookmark individually and keep scrolling and selecting and then we can delete them.
   6. Scrolling and selecting 1000s of bookmark is nearly impossible manually
   7. Same issue is with pc version of brave too.
   8. There is no option in PC Version of brave to select all bookmark simultaneously and delete them.
   9. 

![20211014_153822](https://user-images.githubusercontent.com/65007212/137297665-c2eba5d2-d2ed-42e7-8191-441d600be5c8.jpg)
See in above pic there is no three dot menu to select entire mobile bookmark folder and delete it

![20211014_153927](https://user-images.githubusercontent.com/65007212/137297794-2b234f68-43ae-4b00-8774-2968ba849145.jpg)
See in above pic that is inside mobile bookmark folder i have to select each bookmark individually and keep scrolling through 1080 bookmarks and select them to delete that is nearly imposiible to select that much bookmark individually there is no select all option.

- [ 

What you have shown in :https://github.com/brave/brave-browser/issues/15336#issuecomment-942999128

Is actually a subfolder. If in mobile bookmark folder i select some bookmark and add them to  new subfolder that subfolder can  be deleted. But that is not case whenever user create bookmark in android brave it goes directly into root mobile bookmark folder and when user create bookmark in pc brave it goes to bookmark bar root folder. 
There is no option to delete the root folder and all bookmarks along with it.

## Expected result:

**_Select All Bookmark to Delete Feature:_**
*********************************************************

https://user-images.githubusercontent.com/65007212/138547425-a0667dc0-4c5e-42a6-a711-35c5663b2a33.mp4

## Reproduces how often: 
EASILY REPRODUCED

## Desktop Brave version:
LATEST

## Android Device details:
   - Install type (ARM, x86):ARM
   - Device type (Phone, Tablet, Phablet):PHONE
   - Android version:10

PLEASE IF IT IS IMPOSSIBLE TO DELETE THE ROOT FOLDER ADD OPTION TO SELECT ALL BOOKMARKS AT ONCE AND THEN DELETE THEM.CURRENTLY WE HAVE TO SELECT EACH BOOKMARK INDIVIDUALLY. ON SCREEN MAX 10 BOOKMARK ARE SHOWN AND WE HAVE TO SELECT THEM THEN SCROLL DOWN AND SELECT ANOTHER 10. IN THIS WAY IF WE HAVE TO SELECT 1000s of BOOKMARKS IN ORDER TO DELETE THEM. ITS NEARLY IMPOSSIBLE.

ON PC VERSION ALSO THERE IS NO OPTION TO SELECT ALL BOOKMARKS AT ONE IN ORDER TO DELETE.
THIS ISSUE IS PREVALENT ON BOTH PC AND ANDROID VERSION. ON PC TOO WE HAVE TO SELECT INDIVIDUAL 
BOOKMARK TO DELETE  THEM.

**Comment by tathastu871** [comment]
fuck thats why its a feature request. already moved from brave not using it any more.

**Comment by tortino** [comment]
@rebron 
I think that the original request was not to delete the special folders, but to delete all bookmarks into those folders, without the need to manually select all the bookmarks one by one. So I think it's an error to close this as invalid. It's a perfectly reasonable request, IMHO.

**Comment by bsclifton** [comment]
@tortino you're right - let me update the issue a bit. Original post is worded as deleting the folders themselves, which is just part of Chromium and Brave. But it's absolutely reasonable to provide a way to delete all the bookmarks in those folders

**Comment by bsclifton** [comment]
Updated - thanks for the video @tathastu871 

It would specifically be great to have a "delete all bookmarks in this folder" option here:
![image](https://user-images.githubusercontent.com/4733304/144901197-27152398-fee6-4ad1-956c-44376fd29d48.png)

**Comment by tathastu871** [comment]
is there any progress

**Comment by tathastu871** [comment]
Please wanting this feature for so long.
It wont require much code to just add delete all option and function to delete all bookmarks

**Comment by yashwanth2804** [comment]
@tathastu871  https://community.brave.com/t/replace-all-bookmarks/302923/3 check this if this helps

**Comment by tathastu871** [comment]
KEYBOARD TRICKS DOESNT WORK ON ANDROID. WHY THE HELL ITS SO DIFFICULT TO ASD BASIC FEATURES LIKE THIS TO BROWSER

**Comment by codey-badger** [comment]
Hi @bsclifton, it would be quite handy the option to 'select all' for bookmarks and folders like it's shown in the video in  https://github.com/brave/brave-browser/issues/18766#issue-1026241205

**Comment by tathastu871** [comment]
Seems lot time passed creating this issue, no plans whatsoever

lets say we have to delete 90 bookmark out of 100, if we could quickly select al, then unselect 10 to preserve, then delete rest 90. But currently we have to select 90 individually

**Comment by codey-badger** [comment]
> But currently we have to select 90 individually

even for opening, it's such a pain.
